index.html

```diff
@@ -4737,8 +4737,10 @@ <h2>Data Round Tripping</h2>
       representation as defined in [[XMLSCHEMA11-2]].</p>
 
     <p>The <a>canonical lexical form</a> of an <em>integer</em>, i.e., a
-      <a>number</a> with no non-zero fractional part or a <a>number</a>
-      coerced to <code>xsd:integer</code>, is a finite-length sequence of decimal
+      <a>number</a> with no non-zero fractional part
+      <span class="changed">and an exponent less than or equal to <code>21</code>,</span>
```

Comment:
I think ECMAScript says `< 1e21` (*not* `<= 1e21`) https://tc39.github.io/ecma262/#sec-number.prototype.tofixed
```suggestion
      <span class="changed">and an exponent less than <code>21</code>,</span>
```

Reply:
Yup, mis-read.